src/learner.cc

```diff
@@ -771,7 +772,7 @@ class LearnerImpl : public Learner {
   // name of objective function
   std::string name_obj_;
   // temporal storages for prediction
```

Comment:
Can take the chance and fix this comment as well, I assume it's meant to say "temporary"?